.github/workflows/get_modular_resources.py

```diff
@@ -99,15 +132,54 @@ async def main():
-                "description": model.description,
-                "last_updated": time_now,
-                "url": "https://app.viam.com/ml-model/" + model.public_namespace + "/" + model.name + "/"
-            }
+            if model.ml_model_metadata.model_type & model.ml_model_metadata.model_framework:
+                json_m = {
+                    "id": model.item_id,
+                    "model_id": model.item_id,
+                    "total_organization_usage": int(model.total_organization_usage),
+                    "total_robot_usage": int(model.total_robot_usage),
+                    "description": model.description,
+                    "type": model_type_to_str(model.ml_model_metadata.model_type),
+                    "framework": model_framework_to_str(model.ml_model_metadata.model_framework),
+                    "last_updated": time_now,
+                    "url": "https://app.viam.com/ml-model/" + model.public_namespace + "/" + model.name + "/"
+                }
+            elif model.ml_model_metadata.model_type:
+                json_m = {
+                    "id": model.item_id,
+                    "model_id": model.item_id,
+                    "total_organization_usage": int(model.total_organization_usage),
+                    "total_robot_usage": int(model.total_robot_usage),
+                    "description": model.description,
+                    "type": model_type_to_str(model.ml_model_metadata.model_type),
+                    "framework": "",
+                    "last_updated": time_now,
+                    "url": "https://app.viam.com/ml-model/" + model.public_namespace + "/" + model.name + "/"
+                }
+            elif model.ml_model_metadata.model_framework:
+                json_m = {
+                    "id": model.item_id,
+                    "model_id": model.item_id,
+                    "total_organization_usage": int(model.total_organization_usage),
+                    "total_robot_usage": int(model.total_robot_usage),
+                    "description": model.description,
+                    "type": "",
+                    "framework": model_framework_to_str(model.ml_model_metadata.model_framework),
+                    "last_updated": time_now,
+                    "url": "https://app.viam.com/ml-model/" + model.public_namespace + "/" + model.name + "/"
+                }
+            else:
+                json_m = {
+                    "id": model.item_id,
+                    "model_id": model.item_id,
+                    "total_organization_usage": int(model.total_organization_usage),
+                    "total_robot_usage": int(model.total_robot_usage),
+                    "description": model.description,
+                    "type": "",
+                    "framework": "",
+                    "last_updated": time_now,
+                    "url": "https://app.viam.com/ml-model/" + model.public_namespace + "/" + model.name + "/"
+                }
```

Comment:
Let's clean this up. I haven't run this code, so please test it
```suggestion
            if model.ml_model_metadata.model_framework:
                framework = model_framework_to_str(model.ml_model_metadata.model_framework)
            else:
                framework = ""
            if model.ml_model_metadata.model_type:
                model = model_type_to_str(model.ml_model_metadata.model_type)
            else:
                model = ""

            json_m = {
                "id": model.item_id,
                "model_id": model.item_id,
                "total_organization_usage": int(model.total_organization_usage),
                "total_robot_usage": int(model.total_robot_usage),
                "description": model.description,
                "type": model_type_to_str(model.ml_model_metadata.model_type),
                "framework": model_framework_to_str(model.ml_model_metadata.model_framework),
                "last_updated": time_now,
                "url": "https://app.viam.com/ml-model/" + model.public_namespace + "/" + model.name + "/"
            }
```